doc/DOCUMENTATION.org

```diff
@@ -2078,8 +2078,8 @@ Basic =ido= operations can be done with ~Ctrl~ key:
 
 | Key Binding        | Description                                       |
 |--------------------+---------------------------------------------------|
-| ~C-<return>~       | open a =dired buffer=                             |
-| ~M-<return>~       | open a =dired buffer= in terminal                 |
+| ~C-RET~            | open a =dired buffer=                             |
+| ~M-RET~            | open a =dired buffer= in terminal                 |
```

Comment:
You’re right. Both of these are undefined when I’m a `dired` buffer or trying to use them in the `*scratch*` buffer. I’m not able to find functions that refer to something like “open dired buffer in terminal” (or similar), so I’ll just delete these lines. Will amend.